ONSdigital/eq-author-app
# EAR-1218 fix mutually exclusive options

### What is the context of this PR?

Adding folders removed mutually exclusive options from the qCode page.

It also stopped us from adding qCodes to confirmation questions.

### How to review
- Create a questionnaire
- Make a question with a checkbox that contains a mutually exclusive option
- make sure it appears on the qCode page
- Also make sure you can save a confirmation questions qCode
